src/mame/recfranco/rfjailbrk.cpp

```diff
@@ -0,0 +1,124 @@
+      ||______|  |______|   |__|  |__|  |__|  |__|              |______|   |______|   |______|   |______|   |
+      | ___     ___                                                                            ___   ___    |
+      ||SW1    |SW2                                                                           |SW4  |SW3    |
+      ||__|    |  |            _____    _____    _____    _____    _____               ___    |  |  |  |    |
+      |        |__|           |OKI |   |OKI |   |OKI |   |OKI |   |OKI |   ··   ··    |  |    |__|  |__|    |
+      |          __________   M82C51A  M82C51A  M82C51A  M82C51A  M82C51A JMP11 JMP12 |__<-74HC139M         |
+      |  Xtal   N80C188XL20   ..  ___  ..  ___  ..  ___  ..  ___  ..  ___     __________                    |
+      | 24 MHz  |         | 74HC4024D 74HC4024D 74HC4024D 74HC4024D 74HC4024D| OKI     |   ___   ___        |
+      |         |         |   .. |__|  .. |__|  .. |__|  .. |__|  .. |__|    | M6379   |  |  |  | <-LM380N  |
+      | ___ ___ |_________|   ..       ..       ..       ..       ..         |_________| H11AA1 |__|        |
+74LS20->  ||  |              JMP3     JMP4     JMP8     JMP9     JMP10  ___                                 |
+      ||__||__|  JMP1->..  ..  ..         _____     _____     ___      |SW5      JMP5 ..  JMP6 ..  JMP7 ..  |
+      |   74LS04       .. JMP2 JMP13     |CS82C59A |CS82C59A |HC138A   |  |           ..       ..       ..  |
+      |  ___  ___  _____   _____   _____ |____|    |____|    |__|      |__|        _____   _____   _____    |
+      | |  | |  | |EPROM  |EPROM  DS1644      Xtal               _______          |U39 |  |U40 |  |U41 |    |
+      | |  | |  | |H   |  |L   |  |    |  ___2.4576 ___         |OKI   |          |EMPTY  |EMPTY  |EMPTY    |
+      | |__| |__| |    |  |    |  |    | |  | MHz   |  |        |M82C55A          |    |  |    |  |    |    |
+      2x74HC573AN |    |  |    |  |    | |__|       |__|  ·  ·  |______|          |    |  |    |  |    |    |
+      |           |____|  |____|  |____|74HC4060  74HC74  J3 J4                   |____|  |____|  |____|    |
+      |_________________________________________________test_test___________________________________________|
+
+-JMP3, JMP4, JMP8, JMP9, and JMP10 are for setting up the baud rate of each OKI M82C51A USART (you can
+ configuse 1200, 2400, 4800, 9600, 19200 or 38400 bps.
+
+****************************************************************************************/
+
+#include "emu.h"
+#include "cpu/i86/i186.h"
+#include "machine/i8255.h"
+#include "machine/pic8259.h"
+#include "sound/okim6376.h"
+#include "speaker.h"
+
+namespace {
+
+class rfjailbrk_state : public driver_device
+{
+public:
+	rfjailbrk_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_okim6376(*this, "oki")
```

Comment:
I dislike this naming. I don't need to remember this being 6376 in code, can as well be short-handed to `m_oki` as also the device tag hints.

Reply:
You mean changing _m_okim6376(*this, "oki")_ to _m_oki(*this, "okim6376")_ ??

Thanks!!!!

Reply:
`m_oki(*this, "oki")`

Reply:
Done, thanks!!!!